doc/manual/default.nix

```diff
@@ -13,16 +13,20 @@ let
   options = pkgs.lib.filter (opt: opt.visible && !opt.internal)
     (pkgs.lib.optionAttrSetToDocList systemModule.options);
 
-  optionsXML = builtins.toFile "options.xml" (builtins.unsafeDiscardStringContext
-    (builtins.toXML options));
+  backwardsCompat = { options }: let
```

Comment:
What's the point of making this a function? Wouldn't it make more sense to do

```nix
backwardsCompat =  let 
  optionsXML = ...
in {
 #...
};
```

Reply:
It was made a function so the type signature of that backwardsCompat attribute would be compatible with the type signature of the new nixosOptionsDoc code and both could then be included in the or statement seen on L30 - L32.